tests/test_csc.py

```diff
@@ -381,6 +381,28 @@ async def test_telemetry_loop(self):
                         flush=True, timeout=STD_TIMEOUT
                     )
 
+    async def test_connection_monitor_loop(self):
+        async with self.make_csc(
+            initial_state=salobj.State.STANDBY, config_dir=None, simulation_mode=1
+        ):
+
+            # Enter the Disabled state to construct the connection
+            await salobj.set_summary_state(self.remote, salobj.State.DISABLED)
+            self.assertTrue(self.csc.model.are_clients_connected())
+
+            await asyncio.sleep(2)
```

Comment:
consider using a variable instead of a hard coded number here and in the next sleep.